### PR TITLE
bug with abs/rel view paths, added options(basedir, pretty for debug)

### PR DESCRIPTION
**Pull request by ku3mich**

### CODE REVIEW

**Comment by mavanmanen** [comment]
Seems like you need to make some changes, the appveyor build failed, please update with a fix.

File: Pugzor.Core/PugRendering.cs [hunk 1]
```diff
@@ -5,21 +5,29 @@
...
 using Pugzor.Core.Helpers;
-
+using Microsoft.Extensions.Options;
+
 namespace Pugzor.Core
 {
     public class PugRendering : IPugRendering
     {
         private readonly INodeServices _nodeServices;
+        private PugzorViewEngineOptions Options;
```
Comment (mavanmanen) [inline Pugzor.Core/PugRendering.cs]: please follow the naming format used elsewhere in the program:
`private readonly PugzorViewEngineOptions _options;`

**Comment by ku3mich** [comment]
done, pushed

Regards,
Serhii Kuzmychov

**Comment by ku3mich** [comment]
done, pushed

Regards,
Serhii Kuzmychov

**Comment by ku3mich** [comment]
sorry, did not test ( now all ok

**Comment by mavanmanen** [comment]
Looks good to me 👍

**Comment by dpaquette** [comment]
Thanks so much @ku3mich, this looks good. 

Next time you submit a pull request, can you also create an Issue that describes the bugs or problems that you are fixing? That generally makes it easier for us to keep track of what's happening on the project and what the intention is with your Pull Request.

**Pull request merged**